### PR TITLE
Logging for an exception that will be presented to the client should …

### PR DESCRIPTION
**Pull request by yinlinchen**
…normally be at level ERROR.

- Resolve https://jira.duraspace.org/browse/FCREPO-1874

### CODE REVIEW

**Comment by ajs6f** [comment]
This looks good to me. There are some typos in the same places (e.g. "caught and exception"). Might be worth fixing those too.

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/DocumentStoreExceptionMapper.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
     @Override
     public Response toResponse(final DocumentStoreException e) {
-        LOGGER.debug(
+        LOGGER.error(
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/DocumentStoreExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidQueryExceptionMapper.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
 
     @Override
     public Response toResponse(final InvalidQueryException e) {
-        LOGGER.debug(
+        LOGGER.error(
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidQueryExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidSerializationFormatExceptionMapper.java [hunk 3]
```diff
@@ -43,10 +43,10 @@
     @Override
     public Response toResponse(final InvalidSerializationFormatException e) {
         if (LOGGER.isDebugEnabled()) {
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidSerializationFormatExceptionMapper.java]: Please remove the `LOGGER.isDebugEnabled()` guard.

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidSerializationFormatExceptionMapper.java [hunk 4]
```diff
@@ -43,10 +43,10 @@
...
-            LOGGER.debug("InvalidSerializationFormatException intercepted by InvalidSerializationFormatExceptionMapper"
+            LOGGER.error("InvalidSerializationFormatException intercepted by InvalidSerializationFormatExceptionMapper"
                     + (e.getMessage() != null ? ": " + e.getMessage() : "."), e);
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidSerializationFormatExceptionMapper.java]: Please remove the trailing dump of the raw exception: `e`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/LabelExistsVersionExceptionMapper.java [hunk 5]
```diff
@@ -41,7 +41,7 @@
 
     @Override
     public Response toResponse(final LabelExistsVersionException e) {
-        LOGGER.debug("LabelExistsVersionException intercepted by LabelExistsVersionExceptionMapper"
+        LOGGER.error("LabelExistsVersionException intercepted by LabelExistsVersionExceptionMapper"
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/LabelExistsVersionExceptionMapper.java]: Please remove the trailing dump of the raw exception: `e`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/NodeNotFoundExceptionMapper.java [hunk 6]
```diff
@@ -40,7 +40,7 @@
 
     @Override
     public Response toResponse(final NodeNotFoundException e) {
-        LOGGER.debug(
+        LOGGER.error(
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/NodeNotFoundExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ParamExceptionMapper.java [hunk 7]
```diff
@@ -38,7 +38,7 @@
 
     @Override
     public Response toResponse(final ParamException e) {
-        LOGGER.debug("ParamException intercepted by ParamExceptionMapper: \n", e);
+        LOGGER.error("ParamException intercepted by ParamExceptionMapper: \n", e);
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ParamExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ValueFormatExceptionMapper.java [hunk 8]
```diff
@@ -40,7 +40,7 @@
 
     @Override
     public Response toResponse(final ValueFormatException e) {
-        LOGGER.debug(
+        LOGGER.error(
                 "ValueFormatException intercepted by ValueFormatExceptionMapper: \n", e);
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ValueFormatExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/WebApplicationExceptionMapper.java [hunk 9]
```diff
@@ -39,7 +39,7 @@
 
     @Override
     public Response toResponse(final WebApplicationException e) {
-        LOGGER.debug(
+        LOGGER.error(
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/WebApplicationExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/WildcardExceptionMapper.java [hunk 10]
```diff
@@ -56,7 +56,7 @@ public Response toResponse(final Exception e) {
                     .toResponse((RepositoryException)e.getCause());
         }
 
-        LOGGER.info("Exception intercepted by WildcardExceptionMapper: \n", e);
+        LOGGER.error("Exception intercepted by WildcardExceptionMapper: \n", e);
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/WildcardExceptionMapper.java]: Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`

**Pull request closed without merging**